eppo_client/rules.py

```diff
@@ -40,6 +41,11 @@ def evaluate_condition(
     condition: Condition, subject_attributes: SubjectAttributes
 ) -> bool:
     subject_value = subject_attributes.get(condition.attribute, None)
+    if condition.operator == OperatorType.IS_NULL:
+        if condition.value:
+            return subject_value is None
+        return subject_value is not None
```

Comment:
supernit, optional: could write as `return (subject_value is None) if condition.value else (subject_value is not None)`